worlds/alttp/Bosses.py

```diff
@@ -242,7 +293,7 @@ def place_bosses(world, player: int):
         return  # vanilla bosses come pre-placed
 
     if shuffle_mode in ["basic", "full"]:
-        if world.boss_shuffle[player] == "basic":  # vanilla bosses shuffled
+        if world.boss_shuffle[player].value == "basic":  # vanilla bosses shuffled
```

Comment:
getting .value shouldn't be needed.. why does it look to be needed? Options have comparison methods.